src/ken.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
is this page included in the table of contents? I don't see any updates to the config file

Reply:
Yes, it was added with the previous PR

Reply:
ok